utils/js/LangUtils.ts

```diff
@@ -4,9 +4,9 @@
  *
  * Copyright © 2025 Extremely Heavy Industries Inc.
  */
-import {PlainObject, Thunkable} from '@xh/hoist/core';
+import type {PlainObject, Thunkable, LogSource} from '@xh/hoist/core';
```

Comment:
Is the `import type` option something that we should be using more consistently and rigorously across our Hoist codebase?

Also - was this particular change just a matter of style/best practice, or was this actually a key to solving the problem?

Reply:
> Is the import type option something that we should be using more consistently and rigorously across our Hoist codebase?

I think so, based on what I read.

I don't think this was the root circ dep cause.  This change is more about best practices.

Reply:
I have seen where it resolves circular dependencies and am a big fan.  Not sure if it was needed here.